packages/sse/README.md

```diff
@@ -0,0 +1,174 @@
+# SSE
+
+An unopinionated, minimalist, [standard](https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events)-focused API for managing Server-Sent Events (SSE) in Node.js.
+
+**Node.js**
+
+    const http = require('http');
+    const { SSEService } = require('sse');
+    
+    const sseService = new SSEService()
+    http.createServer((request, response) => {
+      if(request.method === 'GET' && request.url === '/sse')
+        sseService.register(request, response);
+    }).listen(8080);
+    
+    sseService.on('connection', sseId => {
+      sseService.send('hello', sseId); // sends data to a single response
+      sseService.send({event:'user-connected'}); // broadcasts data to all responses
+    });
+
+**Express.js**
+
+    const http = require('http');
+    const express = require('express');
+    const { SSEService } = require('sse');
+    
+    const app = express();
+    const sseService = new SSEService();
+
+    app.get('/sse', sseService.register);
+ 
+    http.createServer(app).listen(8080);
+
+    sseService.on('connection', sseId => {
+      sseService.send('hello', sseId); // sends data to a single response
+      sseService.send({event:'user-connected'}); // broadcasts data to all responses
+    })
+
+# Overview 
+
+Server-Sent Events are entirely managed by an `sseService`. This service guarantees consistency with respect to the [EventSource specification](https://html.spec.whatwg.org/multipage/server-sent-events.html#server-sent-events), provided that the server delegates the request to the `sseService` without sending any headers/data to the response.
+
+The `sseService` allows to send data with fine-grained targeting on the open SSE connections.
+
+# API
+
+ * [Class sse.SSEService](#class-ssesseservice)
+     * [new sse.SSEService()](#new-ssesseservice)
+     * [Class: sseService.SSEID](#class-sseservicesseid)
+        * [sseId.isConnectionActive()](#sseidisconnectionactive)
+        * [sseId.sseService](#sseidsseservice)
+     * [Event: 'clientClose'](#event-clientclose)
+     * [Event: 'connection'](#event-connection)
+     * [Event: 'error'](#event-error)
+     * [sseService.close([cb])](#sseserviceclosecb)
+     * [sseService.register(req, res)](#sseserviceregisterreq-res)
+     * [sseService.resetLastEventId([cb])](#sseserviceresetlasteventidcb)
+     * [sseService.send(opts[, target[, cb]])](#sseservicesendopts-target-cb)
+     * [sseService.unRegister([target[, cb]])](#sseserviceunregistertarget-cb)
```

Comment:
I definitely prefer `unregister`. The reason I went for `unRegister` was because of my IDE considering the former as a spelling error. I went for `unRegister` to prevent the same trouble to the package's users but what the heck, if I start over-engineering spelling this is definitely a sign that something is wrong !

TL;DR : I'll change to `unregister`